dipy/viz/actor.py

```diff
@@ -445,7 +445,7 @@ def streamtube(lines, colors=None, opacity=1, linewidth=0.1, tube_sides=9,
     poly_mapper.ScalarVisibilityOn()
     poly_mapper.SetScalarModeToUsePointFieldData()
     poly_mapper.SelectColorArray("Colors")
-    poly_mapper.GlobalImmediateModeRenderingOn()
+    #poly_mapper.GlobalImmediateModeRenderingOn()
```

Comment:
Hi @RicciWoo, It does not really fix the problem. It would be better to add a vtk version test here. Can you do this change? Thanks

Reply:
I would like to try, but I don't know how to add a test, can you give me some hint?